src/transformStateWithClones.js

```diff
@@ -7,7 +7,32 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
+  const stateCopy = { ...state };
+  const stateCopyArray = [];
```

Comment:
Avoid using data structure names in variable names, it's really confusing

```suggestion
  const stateCopies = [];
```